advisories/0010.md

```diff
@@ -0,0 +1,109 @@
+### CWEs
+CWE-403, CWE-668
+### How to check if your environment is vulnerable
+`runc` version is equal or lower (<=) than `1.1.11-rc1`
+````
+# docker version --format '{{range .Server.Components}}{{if eq .Name "runc"}}{{.Version}}{{end}}{{end}}'
+1.1.11-rc1 <<<<<<<<<<<<<<<
+````
+## How to fix
+### MCR 23.0.x
+Upgrade MCR packages to the fixed 23.0.9-1 version or newer
+* `docker-ee*` packages to:
+  - `23.0.9~4-0` Ubuntu
+  - `23.0.9-4` for RPM package based OSes
+* `containerd.io` package to
+  - `1.6.28~rc.1-2` Ubuntu
+  - `1.6.28-4.1.rc.1.1` for RPM package based OSes
+* `Note:` Restart of the `dockerd` service is `NOT` necessary! 
+#### Ubuntu
+```
+# apt-get install docker-ee=5:23.0.9~4-0* docker-ee-cli=5:23.0.9~4-0* docker-ee-rootless-extras=5:23.0.9~4-0* containerd.io=1.6.28~rc.1-2*
+```
+Check `runc` version:
+```
+# docker version --format '{{range .Server.Components}}{{if eq .Name "runc"}}{{.Version}}{{end}}{{end}}'
+1.1.11-rc1.m1 <<<<<<<<<<<<<<<
+```
+#### RPM package based OSes
+```
+# yum install docker-ee-3:23.0.9-4* docker-ee-cli-1:23.0.9-4* docker-ee-rootless-extras-23.0.9-4* containerd.io-1.6.28-4.1.rc.1.1*
+```
+Check `runc` version:
+```
+# docker version --format '{{range .Server.Components}}{{if eq .Name "runc"}}{{.Version}}{{end}}{{end}}'
+1.1.11-rc1.m1 <<<<<<<<<<<<<<<
+```
+
+For more details please read the [MCR official installation documentation][mcr.docs]
+
+### MCR 20.10.x 
+If you have EOL MCR 20.10.x we `STRONGLY RECOMMEND` to upgrade to the MCR 23.0.9-1.
+But we have an temporary option in case if the MCR version is 20.10.12 (`containerd` 1.6 is used starting from 20.10.12) or newer the CVE can be resolved by upgrade `containerd.io` package to the fixed version.
+`Note` Please be aware that this is an `EXCEPTIONAL` and `TEMPORARY` solution and no new ones are expected in the future.
+#### Ubuntu
+```
+# apt-get install containerd.io=1.6.28~rc.1-2*
+```
+#### RPM package based OSes
+```
+# yum install containerd.io-1.6.28-4.1.rc.1.1*
+```
+
+### Offline deployment
+1. Download packages from [official Mirantis repository][mcr.repos] according your OS and platform
+2. Copy them to offline nodes (using USB Flash for example)
+3. Install these packages on hosts and check `runc` version
+#### Ubuntu
+```
+# dpkg -i docker-ee_23.0.9~4-0* docker-ee-cli_23.0.9~4-0* docker-ee-rootless-extras_23.0.9~4-0* containerd.io=1.6.28~rc.1-2*
+```
+#### RPM package based OSes
+```
+# rpm -i docker-ee-23.0.9-4* docker-ee-cli-23.0.9-4* docker-ee-rootless-extras-23.0.9-4* containerd.io-1.6.28-4.1.rc.1.1*
+```
+
+### Upgrade MCR using Launchpad 
+If your MKE cluster is deployed with Launchpad and configuration correspond to current cluster configuration you can 
+update MCR w/ Launchpad using the follow steps:
+1. Download the [Launchpad][launchpad] v1.5.4 version of later
+2. Run Launchpad apply with `--force upgrade` flag:
+   - `./launchpad-linux-<arch> apply --force-upgrade` 
+3. Check `runc` version on all nodes
```

Comment:
These launchpad steps are ONLY needed if the customer is already using 23.0.9 and needs to upgrade to 23.0.9+security fix.

I think that your text looks good